app/templates/admin/permissions/event-roles.hbs

```diff
@@ -17,18 +17,19 @@
     </tr>
   </thead>
   <tbody>
-    {{#each model.roles as |item|}}
+    {{#each model.roles as |role|}}
       <tr>
-        <td>{{item}}</td>
-        {{#each (range 1 6) as |number|}}
+        <td>{{role}}</td>
+        {{#each (range 1 6) as |index|}}
```

Comment:
@geekyd No it's fine, no need to close this one, as it will fix the issue of assertion error, which was the original objective, I will place the component for loop in a separate commit for clarity :)